bridges/WallpaperStopBridge.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+* WallpaperStopBridge
+* Returns the latests wallpapers from http://www.wallpaperstop.com
+*
+* @name WallpaperStop Bridge
+* @homepage http://www.wallpaperstop.com/
+* @description Returns the latests wallpapers from WallpaperStop
+* @maintainer nel50n
+* @update 2014-11-05
```

Comment:
It's a new Git versioned file, not a really new one.
Anyway, thank you for the code revision.